src/tests/tracing/eventlistener/EventListenerThreadPool.cs

```diff
@@ -98,32 +128,17 @@ static int Main()
                     }
                 }
 
-                // Test this part in Windows only
-                if (TestLibrary.Utilities.IsWindows)
+                if (listener.TPIOPack != listener.TPIOPackGoal &&
+                    listener.TPIOEnqueue != listener.TPIOEnqueueGoal &&
+                    listener.TPIODequeue != listener.TPIODequeueGoal)
```

Comment:
I'm not sure about this test, but there may be cases where several tests run in the same process. Maybe check that the number of events seen is at least the goal in case some other test also raises these events.